include/gazebo_motor_model.h

```diff
@@ -42,6 +42,8 @@
 namespace turning_direction {
 const static int CCW = 1;
 const static int CW = -1;
+const static int CCW_REVERSABLE = 2;
+const static int CW_REVERSABLE = -2;
```

Comment:
As I commented in the other part of the code, I think reversible should be a separate state (possibly a boolean)